src/logger/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
What is the motivation for having on separate cmake package? Seems there's disparity of cmake versions so may be best to having it in same module

Reply:
Fixed.